components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -1035,6 +1036,28 @@ sealed class DownloadAction : BrowserAction() {
     data class RestoreDownloadStateAction(val download: DownloadState) : DownloadAction()
 }
 
+/**
+ * [BrowserAction] implementations related to updating the global state of internet resources to be shared.
+ */
```

Comment:
nit: I think "the global state" is not the correct term as the state is related to one session.

Reply:
Thank you! Will change "global state" with "session state".